egi_notebooks_hub/d4science.py

```diff
@@ -473,7 +473,8 @@ def profile_list(self, spawner):
                 profiles.append(profile)
         if self.extra_profiles:
             profiles.extend(self.extra_profiles)
-        return profiles
+        sorted_profiles = sorted(profiles, key=lambda x: x['display_name'])
```

Comment:
this sorting breaks the `profiles.insert(0, ...)` a few lines above :(, I think it should be ok , but definitely the source of the sorting should better come from d4science

Reply:
I totally agree. But their answer to do the sorting at the source was:

```
manually ordering is not an option, the alphabetic order is fine with us, this is what I got sorting the list you provided, which is acceptable.
   ...
We have no way for sorting at the source level.
```
Another option is that we insist again.
Also, I commented in the PR about the insert:
`I left the "insert" and "append" instructions as they are to be less intrusive and because the general performance will not be affected, but they could be replaced by a single "append".`

Reply:
Let's keep it like it is then

Reply:
You are missing some python beautification for this to be merged

Reply:
What tool/library is it used for this? So I can configure the same in my IDE

Reply:
https://black.readthedocs.io/en/stable/